src/entities/offerings.ts

```diff
@@ -323,6 +328,12 @@ export interface Offerings {
    * Current offering configured in the RevenueCat dashboard.
    */
   readonly current: Offering | null;
+  /**
+   * Retrieves a specific offering by a placement identifier.
+   * For more info see https://www.revenuecat.com/docs/tools/targeting
+   * @param placementId - The placement identifier to retrieve the offering for.
+   */
+  getCurrentOfferingByPlacementId(placementId: string): Offering | null;
```

Comment:
Can we keep it as a simple data interface and populate it where we build instances?
```suggestion
  readonly currentOfferingByPlacementId: Offering | null;
```

Reply:
@tonidero One suggestion I would make here is have this name match the name of the other JS SDK. RN uses 👇

`getCurrentOfferingForPlacement`

Reply:
Actually, that makes more sense... We can move this function to be part of the `Purchases` object, instead of part of the `offerings` interface... I will do that change, which will clean this up